analyzer/src/main/kotlin/Analyzer.kt

```diff
@@ -117,7 +117,7 @@ class Analyzer(private val config: AnalyzerConfiguration, private val labels: Ma
     @JvmOverloads
     fun analyze(
         info: ManagedFileInfo,
-        curationProviders: List<PackageCurationProvider> = emptyList()
+        curationProviders: List<Pair<String, PackageCurationProvider>> = emptyList()
```

Comment:
I would have appreciated some explanations in the commit message about the concept of the ID, what it actually is, i.e. a unique identifier for curation providers to distinguish different providers of the same type.